lib/pure/bitops.nim

```diff
@@ -272,6 +332,10 @@ elif useICC_builtins:
 
 proc countSetBits*(x: SomeInteger): int {.inline, noSideEffect.} =
   ## Counts the set bits in integer. (also called `Hamming weight`:idx:.)
+  runnableExamples:
+    doAssert countSetBits(0b0000_0011'u8) == 2
+    doAssert countSetBits(0b1010_1010'u8) == 4
```

Comment:
again, 2nd line doesn't add anything; there's no edge case here. 1st line is enough

Reply:
It is fine as it is with the existing two examples. No need to change/remove anything here.